src/tests/unit.rs

```diff
@@ -0,0 +1,70 @@
+use crate::tests::comp;
+
+#[test]
+fn unit_hello() {
```

Comment:
what is the `unit_` prefix for here? its pretty obvious that the tests are unit tests by context. also none of the other tests have `unit_` (or any other) prefix

Reply:
It was to make it easier to filter test by name in `cargo test`, anyhow I removed it.